pkg/core/config/config.go

```diff
@@ -228,21 +219,25 @@ func (c *Config) validate() error {
 		return err
 	}
 
+	if !*c.EnableBuiltInFiltering {
+		return errors.New("enableBuiltInFiltering must be true, false is no longer supported")
```

Comment:
Could say "or removed from the config" as well so people know they can delete it from their config files.